.eslintrc.json

```diff
@@ -22,9 +22,11 @@
     "rules": {
       "react/prop-types": [0],
       "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+      "react/jsx-props-no-spreading": "off",
       "react/forbid-prop-types": "off",
       "react/no-array-index-key": "off",
       "jsx-a11y/no-static-element-interactions": "off",
+      "jsx-a11y/click-events-have-key-events": "off",
```

Comment:
This seems like a really bad one for accessibility, but I can't quite get my head around how to solve it properly. Do we need to create `onKeyDown` and `onKeyUp` handlers that mimic whatever the click does?